src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -205,8 +206,16 @@ private void processScannerResult(final QueryManager qm, final Component compone
         LOGGER.debug("Identified policy matches for %d/%d vulnerabilities (scanKey: %s)"
                 .formatted(matchedPoliciesByVulnUuid.size(), syncedVulns.size(), prettyPrint(scanKey)));
 
+        // Log the matched policies with operation mode LOG
+        LOGGER.debug("List of matched vulnerability policies with mode LOG : " + matchedPoliciesByVulnUuid.values().stream().map(matchedPolicy -> matchedPolicy.getName()).toList());
```

Comment:
Let's log this as `INFO` since it's an active decision of users to enable logging for their policy.